coordinator/src/node/event_post_processing.rs

```diff
@@ -0,0 +1,47 @@
+use crate::db;
```

Comment:
🔧 event_post_processing is very generic. Can we make the file name more specific to what it does? E.g. routing_fee.rs